morango/models.py

```diff
@@ -1,75 +1,108 @@
+import json
+
 from django.db import models
 
-from .utils.uuids import UUIDModelMixin
-from .settings import INDEX_FIELDS
+from .utils.uuids import UUIDModelMixin, UUIDField
 
 
 ###################################################################################################
-# METADATA: Additional data to keep track of the state of the system (sessions, counters, etc)
+# APP MODELS: Abstract models from which app models should inherit in order to make them syncable
 ###################################################################################################
+class SyncableModelQuerySet(models.query.QuerySet):
 
+    def update(self, **kwargs):
+        kwargs.update({'_dirty_bit': True})
+        super(SyncableModelQuerySet, self).update(**kwargs)
+    update.queryset_only = True  # Unsure whether django will not place this on manager class by default
 
 
-###################################################################################################
-# APP MODELS: Abstract models from which app models should inherit in order to make them syncable
-###################################################################################################
+class SyncableModel(UUIDModelMixin):
+    """
+    Base model class for syncing. Other models inherit from this class if they want to make
+    their data syncable across devices.
+    """
+    _morango_partitions = {}
+
+    # morango specific field used for tracking model changes
+    _dirty_bit = models.BooleanField(default=True)
 
+    objects = SyncableModelQuerySet.as_manager()
+    # special reference to syncable manager in case 'objects' is overridden in subclasses
+    syncable_objects = SyncableModelQuerySet.as_manager()
 
-class SyncableModel(models.Model):
-    
-    def serialize(self, *args, **kwargs):
+    class Meta:
+        abstract = True
+
+    def save(self, set_dirty_bit=True, *args, **kwargs):
+
+        if set_dirty_bit:
+            self._dirty_bit = True
+        super(SyncableModel, self).save(*args, **kwargs)
+
+    def serialize(self, fields=None, exclude=None, *args, **kwargs):
```

Comment:
If it's hard to pull them during `__init__` to set `_fields_not_to_serialize`, you could instead just override `serialize` on `MorangoMPTTModel` to run super and then delete those keys from the dict before returning it.

Actually, that would probably be the best way to do it in this case, as it then still leaves `_fields_not_to_serialize` for use by the subclasses, to exclude additional fields if needed.